pkg/cache/server/config.go

```diff
@@ -112,9 +112,11 @@ func NewConfig(opts *cacheserveroptions.CompletedOptions, optionalLocalShardRest
 
 	// disable SSA since the cache server should not change objects in any way
 	// note, that this will break when (if) the feature is locked, until then we should be fine
-	if err := utilfeature.DefaultMutableFeatureGate.Set(fmt.Sprintf("%s=false", features.ServerSideApply)); err != nil {
+	// TODO (fgiloux) best path forward to be confirmed.
```

Comment:
Does this need to be resolved before we merge?

Reply:
are you sure that SSA won't alter the cached objects in anyway? If it does then the replication controller will hot loop.

Reply:
thanks!

Reply:
I have added the possibility to deactivate server side apply through parameters in the kubernetes fork. The last commit makes use of it.
If/When the PR for the kubernetes fork merges I can change the go.mod to point back to it instead of my branch.